### PR TITLE
Add v0.5.3 release notes

### PR DESCRIPTION
**Pull request by jielinxu**

### CODE REVIEW

File: release/v0.5.3.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# What's New in v0.5.3
+
+## 2019-11-14
+
+### Enhancements
+
+- Doubled the transmission speed of search results to the client application through the following updates to gRPC:
+  
+  - Optimize messages.
```
Comment (yamasite) [inline release/v0.5.3.md]: Optimized

File: release/v0.5.3.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - Change the API of generated code.
```
Comment (yamasite) [inline release/v0.5.3.md]: Changed

File: release/v0.5.3.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - Remove compression.
```
Comment (yamasite) [inline release/v0.5.3.md]: Removed

File: release/v0.5.3.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+- Doubled the transmission speed of search results to the client application through the following updates to gRPC:
+  
+  - Optimize messages.
+  - Change the API of generated code.
+  - Remove compression. 
+  
+- Python SDK
+
+  - Seperate the storage of search result ids and distances in different arrays, which reduces the API response time.
```
Comment (yamasite) [inline release/v0.5.3.md]: Separated

File: release/v0.5.3.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+sidebar_label: v0.5.3
+---
+
+# What's New in v0.5.3
+
+## 2019-11-14
+
+### Enhancements
+
+- Doubled the transmission speed of search results to the client application through the following updates to gRPC:
```
Comment (yamasite) [inline release/v0.5.3.md]: Past tense or present tense?

**Comment by jielinxu** [comment]
This fixes #155 and #157

**Pull request merged**